osr2mp4/ImageProcess/Animation/easings.py

```diff
@@ -0,0 +1,215 @@
+import math
+
+
+''' 
+	Easing shit - FireRedz
+	copied from my storyboard renderer which is copied from some random gist
+	
+	t is the current time (or position) of the tween.
+	b is the beginning value of the property.
+	c is the change between the beginning and destination value of the property. | basically destination-start
+	d is the total time of the tween.
+'''
+
+def easeLinear(t, b, c, d):
```

Comment:
snake_case ?

Reply:
nah i prefer it like this, might change this in the future